### DISCUSSION TITLE
https://steinkamp.us/posts/2024-06-27_jpl-care-package-from-1983

### DISCUSSION THREAD

**Discussion by giscus** [zsteinkamp/steinkamp.us#14] [Announcements]
# https://steinkamp.us/posts/2024-06-27_jpl-care-package-from-1983

In March 1983, perhaps in response to a parental request for some nerdy materials for a nerdy kid, the Jet Propulsion Laboratory sent a large envelope to my house. Find out what was inside...

https://steinkamp.us/posts/2024-06-27_jpl-care-package-from-1983

**Comment by Dsteinkamp** [comment]
Big smile!
